packages/now-node/src/helpers.ts

```diff
@@ -78,13 +82,15 @@ function status(res: NowResponse, statusCode: number): NowResponse {
 }
 
 function setCharset(type: string, charset: string) {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
You can add a single line for these at the beginning of the file OR if the rule doesn't make sense for us, you can turn it off for the whole repo in `eslintrc.json`